kura/org.eclipse.kura.camel.cloud.factory/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package: org.apache.camel;version="[2.17.0,3.0.0)",
  org.apache.camel.model;version="[2.17.0,3.0.0)",
  org.apache.camel.support;version="[2.17.0,3.0.0)",
  org.eclipse.kura;version="[1.2,2.0)",
+ org.eclipse.kura.camel.bean;version="[1.1.0,2.0.0)",
```

Comment:
Version interval with micro. It is different compared to the following imports. We discussed about that also for cloud stuff.